src/TestStack.White.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 2013
-VisualStudioVersion = 12.0.40629.0
+VisualStudioVersion = 12.0.31101.0
```

Comment:
Seems like you're on Visual Studio Update 4 instead of 5. Either update your VS or don't include this change